include/El/optimization/bfgs.hpp

```diff
@@ -14,12 +14,12 @@
 namespace El {
 
 template< typename T, typename Function, typename Gradient>
-T zoom( const Function& f, const Gradient& gradient, T f0, const DistMatrix<T>& x0, const DistMatrix<T>& p,
+T Zoom( const Function& f, const Gradient& gradient, T f0, const DistMatrix<T>& x0, const DistMatrix<T>& p,
         T alpha_low, T alpha_high, T c1, T c2){
     DistMatrix<T> x_j(x0);
-    DistMatrix<T> g2(p.Height(), 1);
-    while (alpha_high - alpha_low > 10*El::limits::Epsilon<T>()){
-        T alpha = (alpha_low + alpha_high)/2.0;
+    DistMatrix<T> g2(p.Height(), 1, x0.Grid());
+    while (alpha_high - alpha_low > T(10)*El::limits::Epsilon<Base<T>>()){
```

Comment:
Also, for the sake of consistency with the rest of the library, can we use symmetry with braces and put them on a line by themselves?